table/queries/merged_reduced_scans.sql

```diff
@@ -194,14 +196,15 @@ WITH
     IF(is_control, "CONTROL", domain) as domain,
     category,
     "HTTPS" AS source,
-    country,
+    country_name as country,
     netblock,
     organization,
     controls_failed,
     CleanError(error) AS result,
     ClassifyError(error, "HTTPS", success, blockpage, page_signature) as outcome,
     count(1) AS count
   FROM `firehook-censoredplanet.base.https_scan`
+  JOIN `firehook-censoredplanet.metadata.country_names` ON country_code = country
```

Comment:
Use a `LEFT JOIN` instead to be clearer and more correct. If a country name is missing I want it to still show up.

Reply:
done